test/browser/features/support/env.rb

```diff
@@ -28,6 +28,16 @@ def get_error_message id
   skip_this_scenario unless Maze.driver.local_storage?
 end
 
+Before('@skip_ie_8') do |scenario|
+  browser = Maze.config.browser
+  skip_this_scenario unless browser != "ie_8"
+end
+
+Before('@skip_ie_9') do |scenario|
+  browser = Maze.config.browser
+  skip_this_scenario unless browser != "ie_9"
+end
+
```

Comment:
@joshedney just wanted to check you're happy with these statements

Reply:
LGTM!